res/Sectors/M1105/Nooq.txt

```diff
@@ -0,0 +1,450 @@
+2408                      C435523-6 Ni                           { -2 } (740-2) [7347] - -    311  9 Na G4 V           
+2501                      C223663-8 Na Ni Po Cy O:2607           { -2 } (654+2) [3449] - -    520  9 Na K2 V M7 V      
+2507                      D535251-6 Lo                           { -3 } (411+0) [2144] - -    814 16 Na G6 V           
+2603                      C785896-6 Ga Ph Pa Ri                  {  0 } (977+0) [8897] - -    304  8 Na M8 V           
+2607                      AA8A732-C Oc Ri                        {  3 } (B6E+3) [8A9B] - -    310  9 Na M5 V           
+2701                      C441265-5 Lo Po Re He Pz               { -2 } (911+1) [1137] - -  A 304 13 Na F8 V           
+2703                      C7547A7-5 Ag                           {  0 } (66C-5) [8758] - -    220  9 Na G1 III         
+2707                      C535302-8 Lo                           { -2 } (C21+3) [417B] - -    603 10 Na K9 V M6 V      
+2804                      E444331-4 Lo                           { -3 } (821-2) [6162] - -    900  8 Na M1 V           
+2807                      E200000-0 Ba Va                        { -3 } (400+0) [0000] - -    004 13 Na M1 V K2 V      
+2809                      D627213-7 Lo Pz                        { -3 } (711-1) [112C] - -  A 803 12 Na M5 V           
+2810                      E222615-6 Na Ni Po                     { -3 } (B52+2) [7317] - -    900 10 Na M0 V M8 V      
+2901                      C224524-7 Ni                           { -2 } (444+0) [6336] - -    802 11 Na M1 V           
+2904                      A478202-B Lo                           {  1 } (911+2) [437B] - -    500 11 Na F2 V           
+3002                      D698505-6 Ag Ni                        { -2 } (441-4) [6355] - -    722 12 Na M5 V           
+3004                      E888422-7 Ni Pa                        { -3 } (831+2) [1176] - -    203 13 Na M4 V           
+3008                      B966526-A Ag Ni Pr                     {  2 } (947+0) [574C] - -    702  6 Na K8 V K9 V      
+3101                      C210206-7 Lo                           { -2 } (811-1) [6125] - -    514 14 Na M3 V K4 V      
+3102                      E540000-0 Ba De Po He                  { -3 } (700-3) [0000] - -    022 13 Na M1 V M9 V      
+3104                      E337000-0 Ba                           { -3 } (800+0) [0000] - -    003  9 Na G2 V           
+3108                      C458304-A Lo                           {  0 } (F21+1) [237A] - -    713 12 Na G5 V           
+3207                      A777406-C Ni Pa                        {  1 } (932+3) [3539] - -    501 11 Na M9 V           
+0112                      E300000-0 Ba Va                        { -3 } (C00-3) [0000] - -    000 11 Na M6 V M3 V      
+0113 Tsanphual            A988831-9 Ri Ph Pa                     {  1 } (577+2) [9976] - -    402 12 Hc M4 V           
```

Comment:
Tsanphual (Nooq 0113)-- Calculated importance 2 does not match generated importance 1
Tsanphual (Nooq 0113) - CX Calculated acceptance 9 does not match generated acceptance 10

Reply:
OK, I'll fix this now.